nodeshift/provider/client/gpu_types.go

```diff
@@ -1,11 +1,13 @@
 package client
 
 type GPUConfig struct {
-	GPUName  string `json:"gpuName"`
-	Image    string `json:"image"`
-	SSHKey   string `json:"sshKey"`
-	GPUCount int64  `json:"gpuCount,omitempty"`
-	Region   string `json:"region"`
+	GPUName        string `json:"gpuName"`
+	Image          string `json:"image"`
+	SSHKey         string `json:"sshKey"`
+	GPUCount       int64  `json:"gpuCount,omitempty"`
+	Region         string `json:"region,omitempty"`
+	DiskSizeGB     int64  `json:"disk,omitempty"`
```

Comment:
Not sure about syntax, but seems that `DiskSizeGB` field will be optional, but we discussed that it will be better to make it required.